paket.lock

```diff
@@ -1,10 +1,15 @@
 NUGET
   remote: http://www.nuget.org/api/v2
   specs:
+    Dynamitey (1.0.2)
     Expression.Blend.Sdk (1.0.2)
     Fantomas (2.1)
       FSharp.Compiler.Service (>= 2.0.0.8)
     FSharp.Compiler.Service (2.0.0.8)
+    FSharp.Core (4.0.0.1)
```

Comment:
Same here, by the way, adding the `FSharp.Interop.Dynamic` dependency caused it to add FSharp.Core (4.0.0.1). If this is possible to remove, all the better.

@forki, do you have a better option?